l10n_br_zip/models/l10n_br_zip.py

```diff
@@ -1,10 +1,18 @@
 # Copyright (C) 2012  Renato Lima (Akretion)                                  #
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
+import logging
 from odoo import models, fields, api, _
 from odoo.exceptions import UserError
 
 from odoo.addons.l10n_br_base.tools import misc
+try:
+    import pycep_correios
+except ImportError:
```

Comment:
You can add a logger, this is at instalation, so, you may never see this **UserError**

import logging 
_logger = logging.getLogger(__name__)

```suggestion
except ImportError as err:
    _logger.debug(err)
```
you can change the logger level to info or error, or warning